### PR TITLE
Add covariates to subject show

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: src/data_parsing/data_types.jl [hunk 1]
```diff
@@ -393,6 +393,11 @@ function Base.show(io::IO, subject::Subject)
                      observables)
     println(io, "  Observables: $vals")
   end
+  if length(subject.covariates) >= 100:
```
Comment (ChrisRackauckas) [inline src/data_parsing/data_types.jl]:
```suggestion
  if length(subject.covariates) >= 10:
```

Comment (logankilpatrick) [reply src/data_parsing/data_types.jl]: done!

**Comment by ChrisRackauckas** [comment]
Tests fail.

File: src/data_parsing/data_types.jl [hunk 2]
```diff
@@ -393,6 +393,13 @@ function Base.show(io::IO, subject::Subject)
                      observables)
     println(io, "  Observables: $vals")
   end
+  if subject.covariates != Nothing
```
Comment (asinghvi17) [inline src/data_parsing/data_types.jl]: Shouldn't this be `nothing`?

Comment (ChrisRackauckas) [reply src/data_parsing/data_types.jl]: yes, it should be `!== nothing`

**Pull request merged**